about-mock-interviews/about-mocks.md

```diff
@@ -0,0 +1,65 @@
+- Have at least three different interview partners
+- Complete interviews in at least three different topics
+- Use Ada provided questions
+
+After each mock interview, please submit a mock interview retrospective via the Learn checkpoints we have provided you. 
+
+The variety of partners requirement is to encourage you to experience different styles of interviewers and observe different folks' problem solving strategies. The variety of topics requirement is to encourage you to practice multiple topics that may arise when you begin your job search.
+
+## Mock Interview Questions
+
+[**Mock Interview Question Repository**](https://docs.google.com/document/d/1SmKIpGL_z_IXhgLiaGAf3kPc_bApgH1LtIGcXVoKUHE/edit?usp=sharing)
+
+For your five required mock interviews, we ask that you use Ada provided questions from the linkabove. This is to ensure questions are of an appropriate scope and topic. We encourage you to explore non-Ada interview questions in your own time!
+
+**DO NOT VIEW THE PROBLEMS BEFOREHAND** In order to simulate a real interview experience, please do not preview the questions beforehand. 
+
+When you enter the mock interview, you may choose one or more topics for your interviewer to select your question from. The topics are:
+- Classroom Topics
+- Dynamic Programming
+- Linked Lists
+- Binary Search Trees
+- Graphs - Breadth First & Depth First Search
+- Graphs - Dijkstra's
+
+
+## Scheduling Mock Interviews
+Ada provides 90 minute instructor facilitated mock interview sessions every other week for each section. During these sessions, instructors will split you into random pairs based on the topics you would like to interview in that day.
+
+Check your section calendar to view available Ada facilitated mock interview session:
+- [Ligers Calendar](https://calendar.google.com/calendar/u/0?cid=Y18wZmVlNmQxZWFhNzM4YWVlNzM5YTRkZjZmNmRjN2YzNzFmNmExMDkxYWFjMzlkODBhM2YzMjkyNDBlN2I4ZDBlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+- [Tigons Calendar](https://calendar.google.com/calendar/u/0?cid=Y19jYmZjN2VkMGRkMjkxY2M3MGNjMGM1NzljYzkxNzY4MzZhMTQ0MWJlYTk1MWRlMTc4ODc3NGE5ZDA0OTI0MTBmQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+
+If you would like to schedule your own sessions, you are free to coordinate with your fellow Adies and organize alternative sessions.
+
+## Reminders
+- It's **100% okay** not to finish the problem in the allotted time. 
+- Your solution does not have to match the sample solution. There are many ways to solve these problems!
+  - Because we cannot anticipate every possible solution, the sample solution only provides one possible solution
+  - If you are an interviewer and are unsure how to help your interviewee because they are choosing an approach that does not match the sample solution, that's okay! Industry interviewers are sometimes unable or unwilling to provide assistance to the interviewee in a real mock. You may also consider looking up alternative solutions online.
```

Comment:
```suggestion
  - If you are an interviewer and are unsure how to help your interviewee because they are choosing an approach that does not match the sample solution, that's okay! Industry interviewers are sometimes unable or unwilling to provide assistance to the interviewee in a real technical interview. You may also consider looking up alternative solutions online.
```